### DISCUSSION TITLE
Need Help Training LORA (Pagefile size is too small  at 32GB already)

### DISCUSSION THREAD

**Discussion by redkjuegos** [bmaltais/kohya_ss#75] [Q&A]
As the title says, i've been trying to train a LORA network using my local setup.
I have a Ryzen 2600  
16 GB RAM
RTX 3060 12GB

Everytime i try to run the script with the lowest settings it says my pagefile is too small, i've tried from 8 GB to 32 GB  nothing seems to work.
I've restarted between changes etc.

My Batch size is 1, the CPU Cores  1, i dont know what else to change.
Any help would be much appreciated.

Here are my current settings:  
![image](https://user-images.githubusercontent.com/86573938/214981958-30c05e29-7f22-46ee-a286-bc87263850f8.png)

**Comment by iznanka** [comment]
Try uncheck "cash latent" and "enable bucket"

**Comment by Zyin055** [comment]
I got this error with nearly identical hardware after installation. I restarted my computer and no longer get the error.
